lib/getters/cartGetters.ts

```diff
@@ -131,6 +131,13 @@ const getFulfillmentLocation = (item: CartItem, locations): string => {
   return locations.find((location) => location.code === item?.fulfillmentLocationCode)?.name
 }
 
+const getAddToCartTotalPrice = (item: CartItem): Number => {
+  const regular = item.product?.price?.price
+  const special = item.product?.price?.salePrice
+  const qty = item?.quantity ? item?.quantity : 0
```

Comment:
don't calculate prices, totals, etc client side. 
slack message sent with more details on where to get this data

Reply:
I saw your slack message but one doubt I have. CartItem 'total' is giving amount based on total qty for the item if item already added to cart then also it's including old qty.. So, is that correct ?

Reply:
Modified pricing stuff is removed now, will be handled in a separate ticket.